evap/staff/views.py

```diff
@@ -1040,6 +1040,41 @@ def course_copy(request, course_id):
     )
 
 
+@require_POST
+@manager_required
+def create_exam_evaluation(request):
+    evaluation = get_object_from_dict_pk_entry_or_logged_40x(Evaluation, request.POST, "evaluation_id")
+    exam_date = datetime.today()  # request.POST.get("date")
+    if evaluation.is_single_result:
+        raise SuspiciousOperation("Creating an exam evaluation for a single result evaluation is not allowed")
+
+    if evaluation.course.evaluations.filter(name_de="Klausur", name_en="Exam").exists():
+        messages.error(request, _("An exam evaluation already exists for this course."))
+        return HttpResponse()  # 200 OK
+
+    evaluation.weight = 9
+
+    course_evaluation_end_date = exam_date - timedelta(days=1)
+    if evaluation.vote_start_datetime > course_evaluation_end_date:
+        raise SuspiciousOperation("The selected date is before the start date")
```

Comment:
please make this more clear, such as "The exam date is before the start date of the main evaluation"